src/content/docs/agents/go-agent/instrumentation/instrument-go-segments.mdx

```diff
@@ -4,7 +4,7 @@ tags:
   - Agents
   - Go agent
   - Instrumentation
-metaDescription: 'With New Relic APM''s Go agent, you can set up segments, which measure the timing of specific blocks of code in your Golang app.'
+metaDescription: 'With APM's Go agent, you can set up segments, which measure the timing of specific blocks of code in your Golang app.'
```

Comment:
Removing the `''` stops this page erroring, but the error is being thrown for a few other pages as well so you'll need to check all the changed pages that have a pattern like this

```suggestion
metaDescription: With APM's Go agent, you can set up segments, which measure the timing of specific blocks of code in your Golang app.
```

Reply:
Thanks! Will do!